benchmark/scenarios/_request_load.py

```diff
@@ -54,8 +54,7 @@ class RequestLoadScenario(object):
     requests at a specified rate.
 
     :ivar reactor: Reactor to use.
-    :ivar scenario_setup: provider of the interface
-        ``IRequestScenarioSetup``.
+    :ivar scenario_setup: provider of the interface ``IRequest``.
```

Comment:
I think we should rename `scenario_setup` and `scenario_setup_instance` to something that implies that they are for requests. In the test code, `request` is used, can we use a name like that here?